lesson_1.py

```diff
@@ -0,0 +1,27 @@
+def multiplication_table(a, b):
+    for i in range(1, a + 1):
+        for j in range(1, b + 1):
+            print(f"{i} x {j} = {i * j}")
+
+
+def num_input():
+    exit = False
+    while not exit:
+        numbers = input("Введите 2 целых числа через пробел (или q для выхода): ").split()[:2]
+        if numbers[0] == "q" or numbers[0] == "й":
```

Comment:
Такие проверки можно делать через проверку вхождения в массив (список или другой):
```suggestion
        if numbers[0] in ["q", "й"]:
```